repos/system_upgrade/el7toel8/models/rhsminfo.py

```diff
@@ -18,3 +18,5 @@ class RHSMInfo(Model):
     """ Repositories that are enabled on the current system through the subscription-manager. """
     existing_product_certificates = fields.List(fields.String(), default=[])
     """ Product certificates that are currently installed on the system. """
+    status = fields.String(default='')
+    """ RHSM status output. """
```

Comment:
@mreznik wouldn't be better in this case just boolean field like `is_sca`? or it seems it could be even generic "mode" and haviing choices `CAM` and `SCA`. But mdoe could be confusing a little bit for someone, so `is_sca` / `sca_detected` / ... seems to be a better solution. As it is clearly providing info usable by other actors, without need of additional output_parsing in other actors in future (if any actor need it in future).

Reply:
@pirat89 changed...